### DISCUSSION TITLE
Claim clanmates' league pass gold

### DISCUSSION THREAD

**Discussion by mercertom** [SergeyPotapov01/bot_Clash_Royale#16] [Ideas]
Also just scan for button saying "Claim" and get 750 gold x 10 every season.

**Comment by SergeyPotapov01** [comment]
this is basically unrealistic.

to do this, you need to scroll through the clan chat to find this button

**Comment by mercertom** [comment]
Yes. You can scroll to find buttons. Same as how cards are requested. Can
either scroll by sending swipe to emulator or tapping the scroll button
(arrow).
